calico/openstack/test/test_plugin_etcd.py

```diff
@@ -95,14 +109,23 @@ def assertEtcdDeletes(self, expected):
             self.assertEqual(self.recent_deletes, expected)
         self.recent_deletes = set()
 
-    def etcd_read(self, key, recursive=False, timeout=None):
+    def etcd_read(self, key, wait=False, waitIndex=None, recursive=False, timeout=None):##add wait&waitIndex
         """Read from the accumulated etcd database.
         """
         self.maybe_reset_etcd()
 
+        ## Slow down reading from etcd status subtree to allow threads to run more often
+        if wait and key == STATUS_DIR:##
```

Comment:
Comments should begin with a single octothorpe, and the empty comment should be removed.